### DISCUSSION TITLE
Connect to an external ticket system (Like Bugzilla)

### DISCUSSION THREAD

**Discussion by lockered** [amborle/featmap#29] [Ideas]
I have added a feature to featmap to connect a workflow to external ticket system. If there is more interest in this feature I can try to start a pull request (Never done that before).

For a little more insight in the idea:

I have added the possibility to configure a kind of base link in the conf.json (for example: "https://bugzilla.example.com/bugzilla/show_bug.cgi?id=")and add a column in the workflows table, to contain a ticket number. Opening a workflow/goal card, you can enter the number, clicking a button, if you want to. Outside the edit mode the ticket number represents a link to open the ticket system in a new window.
I do not know other ticket systems, but I would think, that modern systems will follow a kind of REST-Semantics, so it should be possible to use the basic idea.

**Comment by amborle** [comment]
Congratulations on adding your own feature to Featmap! Happy to hear that you got around the database issue we discussed previously.

I think this particular feature is a little to specific in order for it to be added to Featmap. In any case, wouldn't a link to the external ticket system in the Description of the card be enough?

**Comment by lockered** [comment]
You identified me correctly. Your tip was correct, but the problem we discussed via email has nothing to do with this feature. I've forgot to say thanks! So better late than never: Thank you!

So for your question: Yes, it would be possible, to add a link via the description of a card but you have to type the complete link. With this addition you only have to type the ticket number and can afterwards directly jump into your ticket system. We use this kind of connection in our scrum tool and it is quite handy.  You can quickly connect the story map or at least a goal to an external system where extensions of customers will be tracked.

I have taken Bugzilla as an example. If you look at OTRS the pattern is identical:
https://otrs.example.com/otrs/customer.pl?Action=CustomerTicketZoom;TicketNumber=
I don't know Jira, but I think their issue tracker will follow a similar pattern.
